tox.ini

```diff
@@ -121,7 +121,7 @@ deps =
 commands =
     coverage --version
```

Comment:
Simulating pytest's behavior manually seemed pretty tough.  I tried it, but gave up.  The pytest folder covers what we need, and it's a small fixture, so I opted to add it here to get 100% coverage.